src/foam/u2/Element.js

```diff
@@ -2128,10 +2128,19 @@ foam.CLASS({
       name: 'attribute',
       value: false
     },
+    {
+      class: 'String',
+      name: 'placeholder'
+    },
     {
       class: 'foam.u2.ViewSpec',
       name: 'view',
-      value: { class: 'foam.u2.TextField' }
+      expression: function (placeholder) {
```

Comment:
TextField has a fromProperty() method which is where it can copy settings out of the property. This should be done there.

Reply:
commit 99ffd1d132b91d24cf78746d7f4657f419fd1cd0 (HEAD -> master, origin/master, origin/HEAD)
Author: Kevin Greer <kgrgreer@gmail.com>
Date:   Tue Jun 4 22:05:38 2019 -0400

    Fix TextField style of setting placeholder.